shell/platform/tizen/tizen_renderer.cc

```diff
@@ -0,0 +1,455 @@
+  EGLint n;
+  // Get the List of EGL framebuffer configuration matches with configAttribs in
+  // list "configs"
+  if (!eglChooseConfig(egl_display_, configAttribs, configs, numOfConfig, &n)) {
+    free(configs);
+    configs = NULL;
+    PrintEGLError();
+    return false;
+  }
+
+  EGLint size;
+  for (int i = 0; i < n; i++) {
+    eglGetConfigAttrib(egl_display_, configs[i], EGL_BUFFER_SIZE, &size);
+    if (bufferSize == size) {
+      egl_config_ = configs[i];
+      break;
+    }
+  }
+  free(configs);
+  configs = NULL;
+  return true;
+}
+
+void TizenRenderer::PrintEGLError() {
+  EGLint error = eglGetError();
+  switch (error) {
+    case EGL_BAD_DISPLAY: {
+      FT_LOGE("EGL_BAD_DISPLAY : Display is not an EGL display connection\n");
+      break;
+    }
+    case EGL_NOT_INITIALIZED: {
+      FT_LOGE("EGL_NOT_INITIALIZED : Display has not been initialized\n");
+      break;
+    }
+    case EGL_BAD_SURFACE: {
+      FT_LOGE("EGL_BAD_SURFACE : Draw or read is not an EGL surface\n");
+      break;
+    }
+    case EGL_BAD_CONTEXT: {
+      FT_LOGE("EGL_BAD_CONTEXT : Context is not an EGL rendering context\n");
+      break;
+    }
+    case EGL_BAD_CONFIG: {
+      FT_LOGE(
+          "EGL_BAD_CONFIG : Config is not an EGL frame buffer configuration\n");
+      break;
+    }
+    case EGL_BAD_MATCH: {
+      FT_LOGE(
+          "EGL_BAD_MATCH : Draw or read are not compatible with context, or if "
+          "context is set to EGL_NO_CONTEXT and draw or read are not set to "
+          "EGL_NO_SURFACE, or if draw or read are set to EGL_NO_SURFACE and "
+          "context is not set to EGL_NO_CONTEXT\n");
+      break;
+    }
+    case EGL_BAD_ACCESS: {
+      FT_LOGE("EGL_BAD_ACCESS : Context is current to some other thread\n");
+      break;
+    }
+    case EGL_BAD_NATIVE_PIXMAP: {
+      FT_LOGE(
+          "EGL_BAD_NATIVE_PIXMAP : A native pixmap underlying either draw or "
+          "read is no longer valid\n");
+      break;
+    }
+    case EGL_BAD_NATIVE_WINDOW: {
+      FT_LOGE(
+          "EGL_BAD_NATIVE_WINDOW : A native window underlying either draw or "
+          "read is no longer valid\n");
+      break;
+    }
+    case EGL_BAD_CURRENT_SURFACE: {
+      FT_LOGE(
+          "EGL_BAD_CURRENT_SURFACE : The previous context has unflushed "
+          "commands and the previous surface is no longer valid\n");
+      break;
+    }
+    case EGL_BAD_ALLOC: {
+      FT_LOGE(
+          "EGL_BAD_ALLOC : Allocation of ancillary buffers for draw or read "
+          "were delayed until eglMakeCurrent is called, and there are not "
+          "enough resources to allocate them\n");
+      break;
+    }
+    case EGL_CONTEXT_LOST: {
+      FT_LOGE(
+          "EGL_CONTEXT_LOST : If a power management event has occurred. The "
+          "application must destroy all contexts and reinitialise OpenGL ES "
+          "state and objects to continue rendering\n");
```

Comment:
```suggestion
          "EGL_CONTEXT_LOST : A power management event has occurred. The "
          "application must destroy all contexts and reinitialise OpenGL ES "
          "state and objects to continue rendering\n");
```
Also do you have any sources for these error strings? I'm asking because different sources state different error reasons for the same error codes. I referred to the latest EGL specification: https://www.khronos.org/registry/EGL/specs/eglspec.1.5.pdf